tests/integration/targets/memory/meta/main.yml

```diff
@@ -1,3 +1,4 @@
 ---
 dependencies:
   - setup_sqlserver_test_plugins
+  - setup_sqlserver
```

Comment:
we'll need to try this out, but I think it will work to remove this from here, and instead add it to `aliases` with a line like this:

```
setup/once/setup_sqlserver
```

that way it will only run once in the whole run; test targets (including the setup targets) run isolated (different ansible executions) but I think installing the module will persist.

Added to dependencies like this, we'd have each target trying to (re)install it.

Reply:
Makes sense, will give it a go in the sanity test fix branch - thnx